edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -308,12 +306,28 @@ iframe.rocket-chat-view {
 .login-register div.toggle-form {
   margin-top: 15px;
   text-align: center;
+
+  h1 {
+    font-size: 45px;
+    color: $vue-black;
+    font-weight: 600;
+  }
+
+  span {
+    color: $vue-black;
+  }
 }
+
 .login-register .action-primary {
   width: 100%;
   display: block;
 }
 
+.login-container {
```

Comment:
Does this work for mobile views too? @luismorenolopera

Reply:
I have left an image as a reference for a 480px screen @Squirrel18

Reply:
But, it works well on mobile views, right?

Reply:
yes